packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed detach issue of dropdown of `search` variants with `isTouchAccessible` prop.
```

Comment:
Because `search` is one of the available variants I'd recommend we rephrase this changelog entry. 

Something similar to this perhaps:

```
* Fixed issue when using `isTouchAccessible` that could cause the dropdown to detach from the input
```

Reply:
Changed cb75c3a